### ISSUE TITLE
Perspective Matrix not consistent with OpenGL or GLM?

### ISSUE THREAD

**Issue by palodequeso** [sebcrozet/nalgebra#88]
I just want to make sure that this is functioning as  you intended it to.

With the following rust code
``` rust
let persp: na::Mat4<f32> = na::PerspMat3::new(800.0 / 500.0, 45.0, 0.0001, 100.0).to_mat();
```

I get the following output:
```
-1.120369, 0,        0,        0
0,         1.792591, 0,        0
0,         0,        1.000002, -0.0002
0,         0,        1,        0
```

With the following glm code in c++
``` c++
glm::mat4 proj = glm::perspective(45.0f, 800.0f / 500.0f, 0.0001f, 100.f);
```

I get the following:
```
1.120369, 0,        0,         0
0,        1.792591, 0,         0
0,        0,        -1.000002, 0,
0,        0,        -1,        0
```

The changes in sign cause my axis to be vertically inverted, such that (0, -1, 0) is up, but I would prefer that (0, 1, 0) be up. I'll gladly write my own perspective matrix function but I just wanted to make sure that this was intended on your end.

**Comment by aepsil0n** [comment]
This has been discussed in #44. There is a recipe on how to use `PerspMat3` to achieve this. I think it is intentional but quite different from the standard CG conventions.

**Comment by sebcrozet** [comment]
Fixed by #175 . We now use standard CG conventions.
